src/components/Divider/__tests__/Divider.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import { colors } from '@magnetis/astro-tokens';
+
+import Divider from '../Divider';
+import type { DividerProps } from '../Divider';
+
+const initialProps: DividerProps = {
+  orientation: 'horizontal',
+  size: 'hairline',
+  type: 'neutral',
+};
+
+describe('Divider', () => {
+  it('renders correctly with default props', () => {
+    const { getByTestId } = render(<Divider {...initialProps} />);
+
+    const divider = getByTestId('Divider.Line');
+
+    expect(divider).toHaveStyle({
+      backgroundColor: colors.transparentFaintClear,
+      height: 1,
+      width: '100%',
+    });
+  });
+
+  it('renders properly when the orientation is vertical', () => {
+    const { getByTestId } = render(<Divider {...initialProps} orientation="vertical" />);
+
+    const divider = getByTestId('Divider.Line');
+
+    expect(divider).toHaveStyle({
+      backgroundColor: colors.transparentFaintClear,
+      height: '100%',
+      width: 1,
+    });
+  });
+
+  it('renders properly when the type is inverse,', () => {
+    const { getByTestId } = render(<Divider {...initialProps} type="inverse" />);
+
+    const divider = getByTestId('Divider.Line');
+
+    expect(divider).toHaveStyle({
+      backgroundColor: colors.transparentBrightClear,
+      height: 1,
+      width: '100%',
+    });
+  });
+
+  it('renders properly when the orientation is vertical, the type is inverse and size of 4 pixels,', () => {
```

Comment:
💅🏾 

```suggestion
  it('renders properly when the orientation is vertical, the type is inverse and size is of 4 pixels,', () => {
```

Reply:
Eu diria que deveria mudar a parte do size para ter o valor que passamos não o valor que é atribuído no estilo
```suggestion
  it('renders properly when the orientation is vertical, the type is inverse and size is thick,', () => {
```

Reply:
Fixed in [61de2ac](https://github.com/magnetis/astro-native/pull/782/commits/61de2acb8a0d6e0340f134e916e88bc4e2aa5242)